### PR TITLE
feat: fix OIDC address field

### PR DESCRIPTION
**Pull request by Bsheepcoder**
Fix: #3001 

OIDC : https://openid.net/specs/openid-connect-core-1_0-errata2.html#AddressClaim

I test with go-oidc，test result：
image

go-oidc key code：
```go
type Address struct {
	Formatted     string `json:"formatted"`
	StreetAddress string `json:"street_address"`
	Locality      string `json:"locality"`
	Region        string `json:"region"`
	PostalCode    string `json:"postal_code"`
	Country       string `json:"country"`
}

type UserInfo struct {
	Sub     string  `json:"sub"`
	Name    string  `json:"name"`
	Email   string  `json:"email"`
	Picture string  `json:"picture"`
	Address Address `json:"address"`
}

var userInfo UserInfo
if err := idToken.Claims(&userInfo); err != nil {
	http.Error(w, "Failed to parse claims", http.StatusInternalServerError)
	return
}

// Output user info as JSON
data, err := json.MarshalIndent(userInfo, "", "  ")
w.Header().Set("Content-Type", "application/json")
w.Write(data)

```

### CODE REVIEW

File: object/user.go [hunk 1]
```diff
@@ -72,6 +72,11 @@ type User struct {
 	Region            string   `xorm:"varchar(100)" json:"region"`
 	Location          string   `xorm:"varchar(100)" json:"location"`
 	Address           []string `json:"address"`
+	Formatted         string   `xorm:"varchar(500)" json:"formatted"`
```
Comment (dacongda) [inline object/user.go:75]: this new field should be in one object call Address, did this pr tested ?

Comment (dacongda) [reply object/user.go:75]: this field will no longer useful. it should be removed now

**Comment by hsluoyz** [comment]
@dacongda plz review

**Review by dacongda — APPROVED** [review]

**Comment by hsluoyz** [comment]
Reverted due to severe bug

**Comment by savely-krasovsky** [comment]
Are there any plans to fix it again properly? Cannot integrate with PhotoPrism due to this bug.

**Comment by dacongda** [comment]
> Are there any plans to fix it again properly? Cannot integrate with PhotoPrism due to this bug.

Set the application's JWT type to JWT-Empty might temporarily solve your problem

**Pull request merged**